core/templates/third-party-imports/ui-codemirror.import.ts

```diff
@@ -17,9 +17,15 @@
  */
 
 window.CodeMirror = require('codemirror');
+window.diff_match_patch = require('diff-match-patch');
+window.DIFF_DELETE = require('diff-match-patch');
+window.DIFF_EQUAL = require('diff-match-patch');
+window.DIFF_INSERT = require('diff-match-patch');
```

Comment:
Add the reason as to why we need these in global scope.

Reply:
undoing the diff-match-patch changes here